aws-redshift-clustersubnetgroup/resource-role.yaml

```diff
@@ -15,6 +15,13 @@ Resources:
             Principal:
               Service: resources.cloudformation.amazonaws.com
             Action: sts:AssumeRole
+            Condition:
+              StringEquals:
```

Comment:
why do we want to add resource subnet group to IAM role condition?

Reply:
this was generated by `mvn clean package`